app/experimenter/templates/base.html

```diff
@@ -62,6 +54,18 @@ <h3 class="pt-2">
               Leave Feedback
             </a>
           </div>
+          <div class="col text-right">
+            <div>
+              <span class="fas fa-user"></span>
+              {{ request.user }}
+            </div>
+
+            <div>
+              <a class="nocolorstyle" href="{% url "home" %}?owner={{ request.user.id }}&amp;archived=on">
+                {{ request.user.owned_experiments.count }} Owned Experiment{{ request.user.experiment_set.count|pluralize:"s" }}
+              </a>
+            </div>
+          </div>
```

Comment:
Were you going to add a similar display for subscribed experiments here?

Reply:
Yeah but taht requires adding a bunch of extra logic so I moved it to https://github.com/mozilla/experimenter/issues/1259